extensions/wikia/LookupContribs/templates/main-form.tmpl.php

```diff
@@ -11,12 +11,12 @@ function lcShowDetails(dbname, mode) {
 		return false;
 	}
 
-	document.location.href = action + '?target=' + username + '&wiki=' + dbname + '&mode=' + mode;
+	document.location.href = action + '?target=' + username + '&wiki=' + mw.html.escape( dbname ) + '&mode=' + mw.html.escape( mode );
 }
 
 $(document).ready(function() {
 	var baseurl = wgScript + "?action=ajax&rs=LookupContribsAjax::axData";
-	var username = '<?= urlencode( $username ) ?>';
+	var username = '<?= htmlspecialchars( $username, ENT_QUOTES ) ?>';
```

Comment:
Once again should be `Xml::escapeJsString`. You're correctly escaping the username when it is added to HTML with `mw.html.escape` so also escaping here will lead to double escaping (safe in general, but could cause issues :)).